jsk_fetch_robot/jsk_fetch_startup/cron_scripts/cron_fetch.conf

```diff
@@ -1 +1,2 @@
 0 3 * * * bash -c ". /home/fetch/ros/melodic/devel/setup.bash && rosrun jsk_fetch_startup update_workspace.sh &> /home/fetch/ros/melodic/update_workspace.log"
+0 3 * * * bash -c ". /home/fetch/ros/melodic/devel/setup.bash && rosrun jsk_fetch_startup storage_warning.sh &> /home/fetch/ros/melodic/storage_warning.log"
```

Comment:
I think `storage_warning.sh` should be executed after `update_workspace.sh`.

This is because storage usage will change by `update_workspace.sh`

How about AM 5:00 (after reboot at AM 4:00)?

Reply:
fixed